prereise/gather/demanddata/transportation_electrification/data_helper.py

```diff
@@ -111,7 +117,13 @@ def generate_daily_weighting(year, area_type="urban"):
     allowable_area_types = {"urban", "rural"}
     if area_type not in allowable_area_types:
         raise ValueError(f"area_type must be one of {allowable_area_types}")
-    data_dir = os.path.join(os.path.dirname(__file__), "data")
+    data_dir = os.path.join(
+        os.path.dirname(inspect.getsourcefile(prereise)),
+        "gather",
+        "demanddata",
+        "transportation_electrification",
+        "data",
+    )
```

Comment:
What is the purpose of the change? Just being curious.

Reply:
Just consistent with the method I used on ECTF, I had some issues with file paths on these files